src/components/ContentPanel/ContentPanel.js

```diff
@@ -4,26 +4,42 @@ import {
 } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
 import { connect } from 'react-redux';
-import { shape } from 'prop-types';
+import { shape, string } from 'prop-types';
 
-import { collectionFlattenedSubTypesSelector } from '../../redux/selectors';
+import { selectedCollectionSubTypeDataSelector } from '../../redux/selectors';
 import SubTypeAccordionsContainer from '../SubTypeAccordion/SubTypeAccordionsContainer';
+import Colors from '../../constants/Colors';
 
-const ContentPanel = ({ collectionFlattenedSubTypes }) => (
-  <ScrollView>
-    <View>
-      <Text style={styles.title}>Details Panel</Text>
-    </View>
-    <SubTypeAccordionsContainer fromContentPanel subTypeData={collectionFlattenedSubTypes} />
-  </ScrollView>
-);
+const ContentPanel = ({
+  collections, selectedCollection, selectedCollectionSubTypeData,
+}) => {
+  const collection = collections[selectedCollection];
```

Comment:
Possibly simpler:

The preview component could be dedicated component that receives a `collectionId` as a manually-drilled prop.

...then, a new, single selector uses `collectionId` (from `ownProps`) to pick the appropriate collection.

Reply:
This is the `ContentPanel` which renders the Selected Collection resourceIds instead of Preview resourceIds.

This comment still probably applies though. How do I pass `ownProps` to a `selector`?